cristin-project/src/main/java/no/unit/nva/cristin/projects/CristinProjectBuilder.java

```diff
@@ -0,0 +1,111 @@
+package no.unit.nva.cristin.projects;
+
+import no.unit.nva.cristin.model.CristinInstitution;
+import no.unit.nva.cristin.projects.model.cristin.CristinFundingSource;
+import no.unit.nva.cristin.projects.model.cristin.CristinPerson;
+import no.unit.nva.cristin.projects.model.cristin.CristinProject;
+import no.unit.nva.cristin.projects.model.cristin.CristinRole;
+import no.unit.nva.cristin.projects.model.nva.Funding;
+import no.unit.nva.cristin.projects.model.nva.NvaContributor;
+import no.unit.nva.cristin.projects.model.nva.NvaProject;
+import no.unit.nva.model.Organization;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+import static no.unit.nva.utils.UriUtils.extractLastPathElement;
+
+public class CristinProjectBuilder {
+
+    private static final Map<String, String> NvaTypesToCristinRoles = Map.of(
+            "ProjectManager",
+            "PRO_MANAGER",
+            "ProjectParticipant",
+            "PRO_PARTICIPANT");
```

Comment:
I guess that we are doing this bi-directionally since we create and retrieve projects. Create an enum that allows this…perhaps only implementing the bi-directionality once it is needed